deploy/terraform-aws/node.sh

```diff
@@ -68,7 +73,9 @@ controllerManager:
     address: 0.0.0.0
 kubernetesVersion: "$k8s_version"
 EOF
+kubeadm config migrate --old-config /tmp/old-kubeadm-config.yaml --new-config /tmp/kubeadm-config.yaml
```

Comment:
Unless there's a reason to keep the old version of the file around I would just update /tmp/kubeadm-config.yaml to have the updated yaml. I don't think we should keep the old configuration around.

Reply:
Hmm, I would prefer to leave it as it is, given the fact we want to support multiple k8s versions (and k8s version could be set in terraform), so I'd prefer to let kubeadm make a decision whether config should be migrated or not